lib/index.js

```diff
@@ -37,6 +55,12 @@ async function run(req, res, fn, onError) {
 
 async function json(req, {limit = '1mb'} = {}) {
   try {
+    if (DEV && req.parsedJson) {
+      return req.parsedJson
+    } else if (DEV && req.jsonError) {
```

Comment:
In theory, there can only be one or the other, but I can switch the order if you like.